src/lib/models/event-history/get-event-categorization.ts

```diff
@@ -20,6 +20,7 @@ export const eventTypeCategorizations: Readonly<
 
   SignalExternalWorkflowExecutionFailed: 'signal',
   SignalExternalWorkflowExecutionInitiated: 'signal',
+  WorkflowExecutionSignaled: 'signal',
```

Comment:
we may want to have a list of categories per event type?

Reply:
Totally agree.